AWS.SAML.psd1

```diff
@@ -12,7 +12,7 @@
     RootModule        = 'AWS.SAML.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '1.0.0'
+    ModuleVersion     = '1.0.1'
```

Comment:
Since we are using semantic versioning can you make this change version 1.1.0.

Reply:
Done!